models/aether_lat-lon/work/input.nml

```diff
@@ -0,0 +1,325 @@
+   output_state_files           = ''
+   output_state_file_list       = 'filter_outputs.txt'
+   output_interval              = 1,
+   output_members               = .true.
+   num_output_state_members     = 20,
+   output_mean                  = .true.
+   output_sd                    = .true.
+   write_all_stages_at_end      = .false.
+   compute_posterior            = .true.
+
+   ens_size                     = 20,
+   num_groups                   = 1,
+   distributed_state            = .true.
+
+   async                        = 4,
+   adv_ens_command              = "./advance_model.csh",
+   tasks_per_model_advance      = 1
+
+   obs_sequence_in_name         = "obs_seq.out.1",
+   obs_sequence_out_name        = "obs_seq.final",
+   num_output_obs_members       = 20,
+   first_obs_days               = -1,
+   first_obs_seconds            = -1,
+   last_obs_days                = -1,
+   last_obs_seconds             = -1,
+   obs_window_days              = -1,
+   obs_window_seconds           = -1,
+
+   inf_flavor                  = 0,                       0,
+   inf_initial_from_restart    = .false.,                 .false.,
+   inf_sd_initial_from_restart = .false.,                 .false.,
+   inf_deterministic           = .true.,                  .true.,
+   inf_initial                 = 1.0,                     1.0,
+   inf_lower_bound             = 0.0,                     1.0,
+   inf_upper_bound             = 1000000.0,               1000000.0,
+   inf_damping                 = 1.0,                     1.0,
+   inf_sd_initial              = 0.0,                     0.0,
+   inf_sd_lower_bound          = 0.0,                     0.0
+   inf_sd_max_change           = 1.05,                    1.05,
+
+   trace_execution              = .false.,
+   output_timestamps            = .false.,
+   output_forward_op_errors     = .false.,
+   write_obs_every_cycle        = .false.,
+   silence                      = .false.,
+
+   allow_missing_clm            = .false.
+   /
+
+
+
+&ensemble_manager_nml
+   /
+
+&assim_tools_nml
+   cutoff                          = 0.2
+   sort_obs_inc                    = .false.
+   spread_restoration              = .false.
+   sampling_error_correction       = .false.
+   adaptive_localization_threshold = -1
+   output_localization_diagnostics = .false.
+   localization_diagnostics_file   = 'localization_diagnostics'
+   print_every_nth_obs             = 0
+   /
+
+&transform_state_nml
+   aether_restart_dirname = 
+      '/Users/raeder/DAI/Manhattan/models/aether_lat-lon/restart_files'
+   variables = 
+      'Temperature',    'neutrals',
+      'O+',             'ions',
+   nblocks_lon = 2
+   nblocks_lat = 2
+   nblocks_lev = 1
+   debug = 0
+   /
+
+&model_nml
+   template_file  = 'filter_input_0001.nc'
+   variables      = 'Temperature',    'QTY_TEMPERATURE',      'NA',  'NA',   'UPDATE',
+                    'Opos',           'QTY_DENSITY_ION_OP',   'NA',  'NA',   'UPDATE'
```

Comment:
Do we need any clamping? maybe for O+?

Reply:
We could certainly make 0.0 be the lower limit for the ion densities and temperatures, since we expect that inflation will sometimes make those be negative.  Or leave that as an exercize for the user.  
Whether it's needed or not depends on whether  Aether and the assimilation setup work well together.  One advantage of not clamping (especially the max values) is that it helps users see when an assimilation is failing, instead of silently correcting itself.